src/pages/Cheon/Login/CheonLogin.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import './CheonLogin.scss';
+import { useNavigate } from 'react-router-dom';
+
+const CheonLogin = () => {
+  const [emailInput, setEmailIdInput] = useState('');
+  const [pwInput, setPwInput] = useState('');
```

Comment:
```suggestion
  const[userInput, setUserInput] = useState({ id: '', password: '' });
```
- 하나의 state로도 관리할 수 있겠네요.